mods/ts/rules/nod-structures.yaml

```diff
@@ -277,6 +277,34 @@ NATECH:
 	SelectionDecorations:
 		VisualBounds: 86, 58, 0, -4
 
+NATMPL:
+	Inherits: ^Building
+	Buildable:
+		Queue: Building
+		BuildPaletteOrder: 200
+		Prerequisites: natech, ~structures.nod
+	ProvidesPrerequisite@buildingname:
+	Valued:
+		Cost: 2000
+	Tooltip:
+		Name: Temple of Nod
+		Description: Provides access to advanced Nod technologies.
+	Building:
+		Footprint: xxxx xxxx xxxx
+		Dimensions: 4,3
```

Comment:
Just try to place them somewhere relevant. Otherwise your unrelated comments will stay there forever and people will never merge it, because there are outstanding remarks.